testsuite/utils/pom.xml

```diff
@@ -109,6 +109,10 @@
                     <groupId>org.slf4j</groupId>
                     <artifactId>slf4j-simple</artifactId>
                 </exclusion>
+                <exclusion>
+                    <groupId>org.eclipse.angus</groupId>
+                    <artifactId>angus-activation</artifactId>
+                </exclusion>
```

Comment:
After the recent changes to this PR, do we still need this?

Reply:
Not anymore